src/app.ts

```diff
@@ -212,3 +213,19 @@ io.use(
 );
 
 socketServer(io);
+
+// Periodically push metrics every 15 seconds to VictoriaMetrics
+if (process.env.ENV === 'prod') {
```

Comment:
Let's not restrict it to just prod. We wanna have this guy publishing for staging/local as well.